tests/locust/locustfile.py

```diff
@@ -0,0 +1,208 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of CERN Open Data Portal.
+# Copyright (C) 2017 CERN.
+#
+# CERN Open Data Portal is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# CERN Open Data Portal is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with CERN Open Data Portal; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Locust file to test CERN Opendata.
+
+Usage:
+
+.. code-block:: console
+
+  $ locust -f tests/locust/locustfile.py --host=http://0.0.0.0
+  [2017-12-19 12:56:37,173] 127.0.0.1/INFO/locust.main: Starting web monitor \
+  at *:8089
+  [2017-12-19 12:56:37,175] 127.0.0.1/INFO/locust.main: Starting Locust 0.8
+  ...
+  $ open http://127.0.0.1:8089
```

Comment:
`open` is macOS specific... e.g. GNU/Linux has `xdg-open`... Perhaps just use `firefox` 😄

Reply:
no `xdg-open` on macOS, I've switched to `firefox`